src/components/img-full/img-full.scss

```diff
@@ -26,11 +26,15 @@
 //Tablet landscape --------------------------------------------------------------------------------------
 @include media-breakpoint-up(lg) {
 
-  .content-column {
+  .DSPreview {
+    margin-left:-3rem;
```

Comment:
Negative values are often code smells.

Why are we negating the margin of the parent? Can we do it in the container?

Reply:
No, this is not possible. In my experience it is quite normal to have a negative margin when you need a full-width image in a container that is not normally full-width.